.travis.yml

```diff
@@ -0,0 +1,57 @@
+language: c
+dist: trusty   # Travis build machine is Ubuntu Trusty 14.04 x64
+sudo: required # A sudo enabled, full VM is used for building instead of a docker container
+
+git:
+  submodules: false
+
+before_install:
+  - sudo apt-get update -qq
```

Comment:
Borrowed from https://github.com/Microsoft/OMS-Agent-for-Linux/blob/020c1b65a849e5e93255af59bfde2f61e25462de/.travis.yml#L9

I wondered about this too - I'll remove this.